src/app/home/home.component.html

```diff
@@ -0,0 +1,91 @@
+<div class="jumbotron">
+  <h1 class="display-3">Library Management System</h1>
+  <p class="lead">
+    We created this Library Management System to sharpen and enhance our web
+    development skills as part of the Smoothstack training.
+  </p>
+</div>
+
+<div class="nav-cards">
+  <div class="admin">
+    <div class="card border-primary mb-3" style="max-width: 25rem;">
+      <div class="card-header">LMS</div>
+      <div class="card-body">
+        <a routerLink="/lms/admin"><h4 class="card-title">Admin</h4></a>
+        <hr class="my-4" />
+        <img
+          style="height: 200px; width: 100%; display: block;"
+          src="../../assets/home-logos/engineer.svg"
+          routerLink="/lms/admin"
+          alt="Card image"
+        />
+        <p class="card-text">
+          The admin has access to perform CRUD operations on all of our data
+          entities and override loans for borrowers.
+        </p>
+        Icons made by
+        <a href="https://www.flaticon.com/authors/itim2101" title="itim2101"
+          >itim2101</a
+        >
+        from
+        <a href="https://www.flaticon.com/" title="Flaticon">
+          www.flaticon.com</a
+        >
+      </div>
+    </div>
+  </div>
+
+  <div class="borrower">
+    <div class="card border-primary mb-3" style="max-width: 25rem;">
+      <div class="card-header">LMS</div>
+      <div class="card-body">
+        <a routerLink="/lms/borrower"><h4 class="card-title">Borrower</h4></a>
+        <hr class="my-4" />
+        <img
+          style="height: 200px; width: 100%; display: block;"
+          routerLink="/lms/borrower"
+          src="../../assets/home-logos/student.svg"
+          alt="Card image"
+        />
+        <p class="card-text">
+          The borrower is able to look through available books and check them
+          out. He can also see his current loans and return books.
+        </p>
+        Icons made by
+        <a href="https://www.flaticon.com/authors/itim2101" title="itim2101"
+          >itim2101</a
+        >
+        from
+        <a href="https://www.flaticon.com/" title="Flaticon">
+          www.flaticon.com</a
+        >
+      </div>
+    </div>
+  </div>
+
+  <div class="librarian">
+    <div class="card border-primary mb-3" style="max-width: 25rem;">
+      <div class="card-header">LMS</div>
+      <div class="card-body">
+        <a routerLink="/lms/librarian"><h4 class="card-title">Librarian</h4></a>
+        <hr class="my-4" />
+        <img
+          style="height: 200px; width: 100%; display: block;"
+          routerLink="/lms/librarian"
+          src="../../assets/home-logos/librarian.svg"
+          alt="Card image"
+        />
+
+        <p class="card-text">
+          The librarian can edit the number of books at a branch and update the
+          details of their branch such as the name, address, and phone number.
```

Comment:
Branch phone numbers aren't in the database.

Reply:
I wasn't very particular with the text in the site. Just wanted to throw stuff in for placeholder. It will probably all need to be changed at the end when we know exactly how the site is working.  if anyone wants to change any of this stuff as we add features feel free!